shell/renderer/api/electron_api_context_bridge.cc

```diff
@@ -303,7 +322,9 @@ v8::MaybeLocal<v8::Value> PassValueToOtherContext(
   // Serializable objects
   blink::CloneableMessage ret;
   {
-    v8::Context::Scope source_context_scope(source_context);
+    v8::Context::Scope error_scope(error_target == BridgeErrorTarget::kSource
+                                       ? source_context
+                                       : destination_context);
     // V8 serializer will throw an error if required
     if (!gin::ConvertFromV8(source_context->GetIsolate(), value, &ret))
```

Comment:
is it ok to use `source_context->GetIsolate()` here or should we use the isolate from the destination context when appropriate? Are they guaranteed to be the same isolate?

Reply:
Technically they're guaranteed to be the same isolate, but for "correctness" I've used the correct context everywhere else, may as well use it here.